examples/run/common/throughput_mt.ipp

```diff
@@ -85,14 +85,18 @@ int throughput_mt(std::string_view description, int argc, char* argv[],
     opts::track_fitting fitting_opts;
     opts::throughput throughput_opts;
     opts::threading threading_opts;
+    opts::logging logging_opts;
     opts::program_options program_opts{
         description,
         {detector_opts, input_opts, clusterization_opts, seeding_opts,
          finding_opts, propagation_opts, fitting_opts, throughput_opts,
-         threading_opts},
+         threading_opts, logging_opts},
         argc,
         argv,
-        logger().cloneWithSuffix("Options")};
+        prelogger->cloneWithSuffix("Options")};
+
+    TRACCC_LOCAL_LOGGER(
+        prelogger->clone(std::nullopt, traccc::Logging::Level(logging_opts)));
```

Comment:
To me this is a good example of why components like `traccc::opts::program_options` should have functional defaults for their logging needs. If we didn't need to supply that object a logger of its own, we could just use `traccc::getDefaultLogger(...)` here conveniently with the now available log level. 🤔

Reply:
I still couldn't agree less with this argument. A class like `program_options` should not be making any decisions about formatting at all, and it certainly should not be printing values to any given output stream in any given format without user intervention. To have these small classes be that opinionated is not good design.

But that's outside the scope of this PR.